trollsched/drawing.py

```diff
@@ -176,10 +184,14 @@ def save_fig(pass_obj,
     plot_parameters = plot_parameters or {}
     with Mapper(**plot_parameters) as mapper:
         mapper.nightshade(pass_obj.uptime, alpha=0.2)
+        for i, p in enumerate(poly):
+            try:
+                c = poly_color[i]
```

Comment:
pep8 requires three letters for variables, did you run pylint @loerum ?
Suggest using `col` instead of `c`!

Reply:
Oh, I see that we have a linter service here, sorry! Strange I thought it should have complained here... ?

Reply:
Hi Adam, I first saw your comments now (don't get notifications). My Python Flymake, does not complain about one letters variables. If it's a simple loop i don't mind one letters, but I will change.

Reply:
Hmm, you comment was it only about the variable c and not i and p ?